be/ut/tx_bulk.c

```diff
@@ -94,8 +106,19 @@ static void be_ut_tx_bulk_test_run(struct m0_be_tx_bulk_cfg    *tb_cfg,
 
 	tb_cfg->tbc_dom = &ut_be->but_dom;
 	rc = m0_be_tx_bulk_init(tb, tb_cfg);
+
 	M0_UT_ASSERT(rc == 0);
-	M0_BE_OP_SYNC(op, m0_be_tx_bulk_run(tb, &op));
+	M0_ALLOC_PTR(op);
+	M0_UT_ASSERT(op != NULL);
+	m0_be_op_init(op);
+
+	m0_be_tx_bulk_run(tb, op);
+	test_work_put(tb, success, ptr);
+	m0_be_op_wait(op);
+
+	m0_be_op_fini(op);
+	m0_free(op);
+
```

Comment:
Can we:
```
M0_BE_OP_SYNC(op, ({m0_be_tx_bulk_run(tb, op); test_work_put(tb, success, ptr);}) );
```

Reply:
Yes, it could be done. I'd prefer to leave it as is because UTs are usually used as examples.